planning/behavior_velocity_planner/src/utilization/util.cpp

```diff
@@ -47,6 +47,40 @@ SearchRangeIndex getPathIndexRangeIncludeLaneId(
   }
   return search_range;
 }
+
+bool setVelocityFrom(const size_t idx, const double vel, PathWithLaneId * input)
```

Comment:
no need to return bool

Reply:
fixed at 
[92446fd](https://github.com/autowarefoundation/autoware.universe/pull/505/commits/92446fd2fea6c8d14a4c820e76ab2e2944cbe8f6)